packages/mdc-dialog/mdc-dialog.scss

```diff
@@ -133,7 +133,6 @@ $mdc-dialog-dark-theme-bg-color: #303030 !default;
     padding: 8px;
   }
 
-  // TODO: Replace with breakpoint variable
```

Comment:
What purpose is it serving being in master? If we are going to replace media queries in all of the packages, we would have a story, and probably search the package for `media-query` or something, no?

I agree with splitting it out. I was just reading through dialog's README to help guide the shape of tabs' README and found these couple of nits with dialog.